lib/meta-boxes.php

```diff
@@ -1,12 +1,11 @@
 <?php
-
 /* Get post objects for select field options */
 function get_post_objects( $query_args ) {
   $args = wp_parse_args( $query_args, array(
-      'posts_per_page' => -1,
-    ) );
+    'post_type' => 'post',
+  ) );
   $posts = get_posts( $args );
-  $post_options = array(" " => null);
+  $post_options = array();
```

Comment:
this was updated from the boilerplate too
that array (old one) seems just wrong tho
:)